### ISSUE TITLE
Do you have `Cannot read property 'addEventListener' of undefined`?

### ISSUE THREAD

**Issue by panlina** [coderaiser/cloudcmd#203]
* **Version** (`cloudcmd -v`): at least since v11.1.0
* **Node Version** `node -v`: v8.11.4
* **Browser name/version**: Chrome 68
* **Used Command Line Parameters**: (no arguments)

Just run `cloudcmd` without any arguments and open the page. The devtool shows:

```
Uncaught (in promise) TypeError: Cannot read property 'addEventListener' of undefined
    at cloudcmd.js:68
    at Generator.next (<anonymous>)
    at n (key.js:79)
    at u (key.js:79)
```

Not sure whether this has any impact, but do you have this?

**Comment by coderaiser** [comment]
No I can't reproduce this, please try to clear site data.
![image](https://user-images.githubusercontent.com/1573141/47642371-2e565f80-db71-11e8-9f0b-4f0d9753c03f.png)

**Comment by panlina** [comment]
I cleared site data and the error remains. I find this error does not exist with Edge. And in chrome it's caused by `navigator.serviceWorker` being `undefined`, at https://github.com/coderaiser/cloudcmd/blob/51ee3773f728fce78377d82a5858869a72ed92d6/client/sw/register.js#L9. Not sure how this happens yet. I'll debug again and find why this happens for me.

**Comment by coderaiser** [comment]
if `navigator.serviceWorker` is `undefined` it should exit function and do not try to register itself. 
You can try to update chrome to `v70`.

**Comment by panlina** [comment]
I'm already using v70. Now from my debug this issue is not related to cloudcmd. I'm using a minimum server serving like `.use((req, res) => res.send("<script>console.log(navigator.serverWorker);</script>")`. This issue happens when it's served from a remote server without https and browsed from Chrome. I did a search and found https://www.chromestatus.com/feature/4835970390163456. It's probably the cause.

**Comment by coderaiser** [comment]
I understood what are you talking about. I didn't test this case for a long time, just tried it and I managed to reproduce this bug, thank you for explanation :).

**Comment by coderaiser** [comment]
Fixed with b782c0a, landed in [v11.7.1](https://github.com/coderaiser/cloudcmd/releases/tag/v11.7.1) 🎉
